src/main/java/race/car/ExtractWinner.java

```diff
@@ -0,0 +1,42 @@
+package race.car;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExtractWinner {
+
+    public static List<CarRaceMoveInfo> ExtractRun(Car car) {
```

Comment:
메서드가 대문자로 시작하고 있네요 😭 
```suggestion
    public static List<CarRaceMoveInfo> extractRun(Car car) {
```